components/cd-tag/cd-tag.css

```diff
@@ -0,0 +1,75 @@
+/**
+ * Base component styles for the collection of tags
+ */
+.cd-tags {
+  /* Override container class if you wish to provide specific styling to all tags */
+}
+
+/**
+ * Individual tag
+ *
+ * - margin is set in `rem` to ensure that multiple tag sizes visually line up
+ * - all other units are `em` so that setting `font-size` can scale them easily
+ */
+.cd-tag {
+  display: inline-block;
+  margin: 0.25rem;
+  padding: 0 .25em;
+  background-color: #5090cd; /* default is up for discussion */
+  color: #fff;
```

Comment:
Colours should use existing colour variables as a CSS custom property for consistency https://github.com/UN-OCHA/common_design/blob/cd-226-design-system/sass/cd/_cd-variables.scss#L47